### ISSUE TITLE
LC_CTYPE=en_US.UTF-8 causes warnings in sct_download_data

### ISSUE THREAD

**Issue by kousu** [neuropoly/spinalcordtoolbox#3128]
Seen in https://github.com/neuropoly/spinalcordtoolbox/runs/1606746772?check_suite_focus=true and again in https://github.com/neuropoly/spinalcordtoolbox/runs/1608360364?check_suite_focus=true. Only on CentOS on Github Actions? Which is weird...because it's a docker image so it should be identical to Travis :haircut_man: ?

There's some locale fiddliness. Given that Travis sets `LC_ALL=en_US.UTF-8`, but not `LC_CTYPE=en_US.UTF-8` like Github Actions does, I suspect that's the problem.

```
Trying URL: https://github.com/sct-data/PAM50/releases/download/r20201104/PAM50-r20201104.zip
Downloading: PAM50-r20201104.zip
--- Logging error ---
Traceback (most recent call last):
  File "/github/home/sct_dev/python/envs/venv_sct/lib/python3.6/logging/__init__.py", line 996, in emit
    stream.write(msg)
UnicodeEncodeError: 'ascii' codec can't encode character '\u201c' in position 42: ordinal not in range(128)
Call stack:
  File "/github/home/sct_dev/spinalcordtoolbox/scripts/sct_download_data.py", line 168, in <module>
    res = main()
  File "/github/home/sct_dev/spinalcordtoolbox/scripts/sct_download_data.py", line 160, in main
    install_data(url, dest_folder, keep=arguments.k)
  File "/github/home/sct_dev/spinalcordtoolbox/download.py", line 141, in install_data
    logger.warning("Removing existing destination folder \u201c%s\u201d", dest_folder)
Message: 'Removing existing destination folder \u201c%s\u201d'
Arguments: ('/github/home/sct_dev/data/PAM50',)
```

I haven't tried to reproduce this yet. It should be relatively easy though:

```
LC_CTYPE=en_US.UTF-8 sct_download_data -d PAM50
```

**Comment by kousu** [comment]
Weirdly, CentOS was fine here https://github.com/neuropoly/spinalcordtoolbox/runs/1606814542?check_suite_focus=true

What's going on?

**Comment by joshuacwnewton** [comment]
https://github.com/neuropoly/spinalcordtoolbox/blob/6c18169f2d2ea8f7b43438c5f505a7fb0eea83d7/spinalcordtoolbox/download.py#L141

Probably easiest just to remove the offending unicode characters, as they don't seem necessary at all.

```diff
- logger.warning("Removing existing destination folder “%s”", dest_folder) 
+ logger.warning("Removing existing destination folder '%s'", dest_folder)
```

**Comment by kousu** [comment]
Yeah definitely!
Smart quotes, the bane of wordprocessing.

**Comment by kousu** [comment]
hmm i might just fix this in #3125 even though it's off-topic; it's blocking that.

**Comment by kousu** [comment]
Oh I just realized this wasn't actually a crash, it was a warning:

https://github.com/python/cpython/blob/bf108bb21e1d75e30bd17141cc531dd08a5e5d0c/Lib/logging/__init__.py#L1000-L1011

The actual crash I saw on https://github.com/neuropoly/spinalcordtoolbox/runs/1606746772?check_suite_focus=true was

```
ls: cannot access 'bin': No such file or directory
```

I thought the unicode problem was causing the crash by preventing proper unpacking of `sct_download_data -d sct_binaries_linux` but maybe it was something else?

**Comment by kousu** [comment]
Addressed in #3135.

The `ls` thing is a different bug. I'll open a new issue for it if I need to.
